apps/kilocode-docs/docs/advanced-usage/cloud-agent.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Cloud Agents
+sidebar_label: Cloud Agents
+---
+
+# Cloud Agents
+
+Cloud Agents let you run Kilo Code in the cloud from any device, without relying on your local machine. They provide a remote development environment that can read and modify your GitHub repositories, run commands, and auto-commit changes as work progresses.
+
+---
+
+## What Cloud Agents Enable
+
+- Run Kilo Code remotely from a browser  
+- Offload compute-heavy tasks to the cloud  
```

Comment:
```suggestion
```

I'd maybe skip this blurb until we know what instance sizes we'll supply. The one we're launching with is a relatively small single vcpu instance.